gubernator/view_build.py

```diff
@@ -127,17 +160,10 @@ def build_details(build_dir):
                   skipped: [name...],
                   passed: [name...]}
     """
-    started_fut = gcs_async.read(build_dir + '/started.json')
-    finished = gcs_async.read(build_dir + '/finished.json').get_result()
-    started = started_fut.get_result()
-    if finished and not started:
-        started = 'null'
-    if started and not finished:
-        finished = 'null'
-    elif not (started and finished):
-        return
-    started = json.loads(started)
-    finished = json.loads(finished)
+    started, finished = normalize_metadata(
+        gcs_async.read(build_dir + '/started.json'),
+        gcs_async.read(build_dir + '/finished.json')
+    )
 
     junit_paths = [f.filename for f in view_base.gcs_ls_recursive('%s/artifacts' % build_dir)
```

Comment:
this shouldn't happen if started & finished are None

Reply:
ACK